src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftOAuth2SecurityRealm.java

```diff
@@ -257,11 +258,13 @@ public void initialize(HttpRequest request) throws IOException {
                                 }
                             });
                     GenericUrl url = new GenericUrl(serverPrefix + "/oapi/v1/users/~");
-
+                    
+                    // do self-sar here to see if you have the "admin" or "view" verb in the "build.openshift.io" api group on the "jenkins" resource 
```

Comment:
Also, as it happens the `oc` in the jenkins image is old enough that it does not have `oc policy can-i`.

@bparees FYI - it may be time to update the `oc` binary in the Jenkins image.